rdflib/compare.py

```diff
@@ -290,7 +292,7 @@ def _initial_color(self):
 
     def _individuate(self, color, individual):
         new_color = list(color.color)
-        new_color.append((len(color.nodes)))
+        new_color.append(tuple([len(color.nodes)]))
```

Comment:
On Mon, 10 Aug 2015 10:33:29 -0700, Jörn Hees <notifications@github.com> said:


    > could also be written as
    > `new_color.append((len(color.nodes),))`, which is a bit faster,
    > but maybe premature optimization:

Creating a value with the right datatype instead of creating one with
the wrong datatype and then converting it isn't optimisation, it's
just doing things correctly.

Or so I think anyways...

-w